src/plugins/saved_objects/public/saved_object/saved_object.test.ts

```diff
@@ -246,7 +245,7 @@ describe('Saved Object', () => {
         return createInitializedSavedObject({ type: 'dashboard' }).then((savedObject) => {
           savedObjectsClientStub.create = jest.fn().mockImplementation(() => {
             expect(savedObject.isSaving).toBe(true);
-            return Bluebird.reject('');
+            return Promise.reject();
```

Comment:
What is the difference between reject a promise `''` in Bluebird and here `reject()`?

Reply:
There is no difference, it's just that bluebird's first parameter is the reject reason. And it is mandatory. So I just removed the message, as it simply does not exist